AdyenUI/ColorScheme.swift

```diff
@@ -0,0 +1,110 @@
+//
+// Copyright (c) 2025 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import UIKit
+
+public struct ColorScheme: Equatable {
+
+    public var background: UIColor
+    public var container: UIColor
+    public var primary: UIColor
+    public var textOnPrimary: UIColor
+    public var action: UIColor
+    public var destructive: UIColor
+    public var textOnDestructive: UIColor
+    public var disabled: UIColor
+    public var textOnDisabled: UIColor
+    public var outline: UIColor
+    public var text: UIColor
+    public var textSecondary: UIColor
+
+    // MARK: - Initializers
+
+    // A static default ColorScheme
+    public static var `default`: ColorScheme {
```

Comment:
What's the reason it to be calculated every time we access it?

Reply:
I was thinking what if there is any preset value for the colors 🤔 but yes having stored property makes more sense here. Thanks for this :)